packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -119,11 +126,11 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
         it('@release should execute "eth_estimateGas" for contract call', async function() {
             const expectedRes = `0x${(400000).toString(16)}`;
             const estimatedGas = await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS, [{
-                to: mirrorContract.evm_address,
+                to: basicContract.evm_address,
```

Comment:
Q: Why are we changing the contract from `mirrorContract` to `basicContract`?

Reply:
The data used to estimate the gas is the encoded ping function of the basic contract, not the mirror contract.  It's represented by the literal below BASIC_CONTRACT_PING_CALL_DATA.  When the basic contract is used the expected estimated gas is returned.

Reply:
Updated the PR description.   Here is the contents of the basic contract: 
```
//SPDX-License-Identifier: Unlicense
pragma solidity ^0.8.0;

contract Basic {

    constructor() {}

    function ping() public pure returns (int) {
        return 1;
    }

    function destroy() public {
        selfdestruct(payable(msg.sender));
    }

    receive() external payable {}
}
```
The encoded ping function:
```
const BASIC_CONTRACT_PING_CALL_DATA = '0x5c36b186';
```
Reference: https://www.moesif.com/solidity-abi-hex-decoder/decode


So the test should use the basic contract not the mirrornode contract:
```
            const estimatedGas = await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS, [{
                to: basicContract.evm_address,
                data: BASIC_CONTRACT_PING_CALL_DATA
            }], requestId);
```